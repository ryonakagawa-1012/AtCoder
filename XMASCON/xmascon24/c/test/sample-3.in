15
2 3 6 11 23 47 106 235 551 1301 3159 7741 19320 48629 123867
3
3 3 3
1 1 2 3 5 7 11 15 22 30 42 56 77 101 135 176 231 297 385 490 627 792 1002 1255 1575 1958 2436
