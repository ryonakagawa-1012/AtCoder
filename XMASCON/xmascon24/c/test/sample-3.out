205001 2733669 22444763 8201007 115532895 978118598 182867184 683781124 283511483 82010070 135215245 487076637 271695954 473451928 217234716 349364028 711882268 472200539 360235417 248161311 666997398 886133207 465643853 863287257 136771343 714472215 407369010
